src/main/java/taxi/controller/driver/LogOutController.java

```diff
@@ -0,0 +1,18 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = "/logout")
+public class LogOutController extends HttpServlet {
```

Comment:
I'm not 100% sure about the importance of the class name, but in the task the class is called LogoutController